src/Microsoft.Azure.WebJobs.Host/Bindings/SupportsRetryAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host
```

Comment:
Recommend moving this to the root folder, peer to RetryAttribute (not in Bindings folder)